docs/guides/quick-start/handling-the-payment.md

```diff
@@ -81,50 +85,50 @@ In order to provide a more common interface, we advise to use the **optional** p
 
-You might need more data / persist data along with the customer which is either - not available when calling the endpoint - or - not supposed to be transmitted at that point - such as saved payment credentials, access tokens or secrets. This data should be transmitted through separate endpoints or be part of the plugin configuration.
+You might need more data/ persist data along with the customer, which is either unavailable when calling the endpoint or not supposed to be transmitted at that point, such as saved payment credentials, access tokens, or secrets. This data should be transmitted through separate endpoints or be part of the plugin configuration.
 
 ## Payment Flows
 
-Depending on the payment method, the user can flow can differ. There are two major ways to perform a payment.
+Depending on the payment method, the user flow can differ. There are two major ways to perform a payment.
 
 ### Synchronous payment
 
-In the synchronous case, the endpoint simply triggers an action that handles the payment \(e.g. an external payment API\) and receives an immediate response.
+In the synchronous case, the endpoint simply triggers an action that handles the payment \(e.g., an external payment API\) and receives an immediate response.
 
 ### Asynchronous payment
 
-In that case, the user flow is different, because the user gets redirected to an external website to perform the payment \(e.g. entering detailed information, logging in etc.\). Shopware will create a JWT \(JSON web token\) containing transaction-related information including:
+In the asynchronous case, the user flow is different. Here the user gets redirected to an external website to perform the payment \(e.g., entering detailed information, logging in etc.\). Shopware will create a JWT \(JSON Web Token\) containing transaction-related information including:
 
-* transaction identifier
-* payment method identifier
-* finish page URL
-* error page URL
+* Transaction identifier
+* Payment method identifier
+* Finish page URL
+* Error page URL
 
-#### Redirect to payment provider / gateway
+#### Redirect to payment provider/ gateway
 
-All this information will be assembled into a **return URL** for the external payment provider containing the token as a parameter `_sw_payment_token`. Together with this **return URL** \(neither the `finishUrl` nor the `errorUrl` - it's a Shopware endpoint\), Shopware will redirect your call to the external payment endpoint to let it conduct the payment.
+All the above transaction-related information will be assembled into a **return URL** for the external payment provider containing the token as a parameter `_sw_payment_token`. Together with this **return URL** \(neither the `finishUrl` nor the `errorUrl` - it's a Shopware endpoint\), Shopware will redirect your call to the external payment endpoint to let it conduct the payment.
```

Comment:
Please remove this information in brackets : (neither the `finishUrl` nor the `errorUrl` - it's a Shopware endpoint\)